sdk/core/version-overrides-matrix.json

```diff
@@ -0,0 +1,19 @@
+{
+  "matrix": {
+    "Agent": {
+      "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" }
+    },
+    "JavaTestVersion": "1.17",
+    "AZURE_TEST_HTTP_CLIENTS": "netty",
+    "TestGoals": "surefire:test",
+    "VersionOverrides": [
+        "jackson_2.10",
+        "jackson_2.11",
+        "jackson_2.12",
+        "reactor_3.5",
+        "reactor_3.4"
+    ],
+    "AZURE_VERSION_OVERRIDE_TESTS": true,
+    "TestFromSource": true
```

Comment:
Do we always want to be testing from source for version validation? For Core this will add 5 ~20-30 minute pipelines that need to complete to pass CI. Thoughts on making this configurable where CI check-in only validates the libraries themselves and then during the test pipeline/scheduled pipeline it uses from source. I just worry that this could massively slow down the ability to check in code

Reply:
moved to live tests and runs in the release pipeline for now.